src/components/templates/CreateResourceMessages.jsx

```diff
@@ -1,15 +1,18 @@
 // Copyright 2019 Stanford University see LICENSE for license
 
 import React from 'react'
-import PropTypes from 'prop-types'
+import { useSelector } from 'react-redux'
+import { createTemplateMessages } from 'selectors/flashSelectors'
 
 /**
  * An info message that is populated when retrieving a resource template
  */
-const CreateResourceMessages = (props) => {
-  if (props.messages.length === 0) return null
+const CreateResourceMessages = () => {
+  const messages = useSelector(state => createTemplateMessages(state))
```

Comment:
Recommend calling the selector `findCreateTemplateMessages` because `createTemplateMessages` sounds like an action.

Reply:
Yes, I've made this change.